depthai_helpers/app_manager.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+initEnv = os.environ.copy()
+
+
+class App:
+    def __init__(self, appName, appPath=None, appRequirements=None, appEntrypoint=None):
+        self.appName = appName
+        self.appPath = appPath or Path(__file__).parent.parent / "apps" / self.appName
+        self.venvPath = self.appPath / "venv"
+        self.appPip = str(self.venvPath / "bin" / "pip")
+        self.appInterpreter = str(self.venvPath / "bin" / "python")
+        self.appRequirements = appRequirements or self.appPath / "requirements.txt"
+        self.appEntrypoint = appEntrypoint or self.appPath / "main.py"
+
+    def createVenv(self, force=False):
+        try:
+            subprocess.check_call([sys.executable, '-m', 'venv', '-h'], env=initEnv, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
Need to put all occourences of sys.executable in codebase between quotes, otherwise it won't work if there's a whitespace in path

Reply:
@VanDavv @szabi-luxonis 
Check call AFAIK takes arguments as is - the quoting here shouldn't make a difference. If `sys.executable` has spaces it'll still be sent correctly as "argv0".

Quoting is a shell concept to correctly determine what goes to eg "argv0", "argv1", ...

Reply:
Indeed, the subprocess is not called in shell context, e.g. `shell=True` so it's not needed 👍🏻